cljfmt/test/cljfmt/test_util/common.cljc

```diff
@@ -0,0 +1,16 @@
+(ns cljfmt.test-util.common
+  (:require [clojure.string :as str]
+            [cljfmt.core :refer [reformat-string]]))
+
+(defn assert-reformats-to
+  ([msg in-lines expected-lines]
+   (assert-reformats-to msg in-lines {} expected-lines))
+  ([msg in-lines options expected-lines]
```

Comment:
I think I'd prefer to have the options last. I understand why you put the `expected-lines` there, but I think the argument list is more predictable if the positioning doesn't change.

Reply:
I see your point and realize I don't feel too strongly about my preference.